source/elements/dpcpp/source/index.rst

```diff
@@ -90,110 +80,65 @@ required if covered by newer C++ or SYCL versions directly.
-   Additional atomic-related queries are defined in Table 4.19, and some changes to fences and barriers are reflected
-   in Section 4.17.1 (both in the `SYCL 2020 Provisional Specification`_).
--  Kernel function type attributes - definition of kernel attributes as function type attributes that allows
-   them to be applied to lambdas.  Definition of some core attributes.  Section 5.7 of the
-   `SYCL 2020 Provisional Specification`_.
--  In-order queues - defines simple in-order semantics for queues, to simplify common coding patterns.
-   Section 4.6.5 of the `SYCL 2020 Provisional Specification`_.
--  Math array - contiguous fixed-size portable container.  Section 4.16.3 of the
-   `SYCL 2020 Provisional Specification`_.
--  Optional lambda name - removes requirement to manually name lambdas that define kernels.
-   Simplifies coding and enables composability with libraries.  Lambdas can still be manually named, if
-   desired, such as when debugging or interfacing with a ``sycl::program`` object.
-   Section 4.14.2 of the `SYCL 2020 Provisional Specification`_.
--  Queue shortcuts - defines kernel invocation functions directly on the queue classes, to simplify code patterns
-   where dependencies and/or accessors do not need to be created within the additional command group scope.  Reduces
-   code verbosity in some common patterns.  Section 4.6.5 of the `SYCL 2020 Provisional Specification`_.
--  Required work-group size - defines an attribute that can be applied to kernels (including lambda definitions of kernels)
-   which signals that the kernel will only be invoked with a specific work-group size.  This is an optimization attribute
-   that enables optimizations based on additional user-driven information.  Section 5.7 of the
-   `SYCL 2020 Provisional Specification`_.
--  Standard layout relaxed - removes the requirement that data shared by a host and device(s) must be C++ standard layout
-   types.  Requires device compilers to validate layout compatibility. Section 4.14.4 of the `SYCL 2020 Provisional Specification`_.
--  Unified Shared Memory (USM) - defines pointer based memory accesses and management interfaces. Provides
-   the ability to create allocations that are visible and have consistent pointer values across both
-   host and device(s).  Different USM capability levels are defined, corresponding to different levels
-   of device and implementation support.  Section 4.8 of the `SYCL 2020 Provisional Specification`_.
-
+The `SYCL 2020 Specification`_ describes the SYCL APIs and language.  DPC++ extensions on top of SYCL
+are described in the `SYCL Extensions`_ repository.
 
 A brief summary of the extensions is as follows:
 
 -  Accessor properties - compile-time accessor properties that are visible to the compiler.
+-  Device-side assert - allows device code to call *assert()*, which when triggered prints a message and aborts the host application.
```

Comment:
Now that you removed "device-side assert" from the table, you should remove this bullet point also.

Reply:
Oops.  Thanks.
Fixed.